File: LQUnionPayDemo/LQUnionPayDemo/ViewController.m
//
//  ViewController.m
//  LQUnionPayDemo
//
//  Created by LiuQiqiang on 2018/11/28.
//  Copyright © 2018 LiuQiqiang. All rights reserved.
//

#import "ViewController.h"
#import "LQUnionPay.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    UIButton *button = [UIButton buttonWithType:(UIButtonTypeCustom)];
    
    button.backgroundColor = [UIColor greenColor];
    button.frame = CGRectMake(40, 100, 100, 40);
    [button addTarget:self action:@selector(testAction) forControlEvents:(UIControlEventTouchUpInside)];
    [self.view addSubview:button];
    
    NSLog(@"%d", [LQUnionPay isUnionAppInstall]);
}

- (void) testAction {
    [LQUnionPay startPayWithTN:@"" onViewController:self resultHandler:^(LQUnionPayResult result, NSDictionary *info) {
        
        NSLog(@"%lu --- %@", (unsigned long)result, info);
    }];
}
@end

File: LQAliSDKDemo/LQAliSDKDemo/ViewController.swift
//
//  ViewController.swift
//  LQAliSDKDemo
//
//  Created by Artron_LQQ on 2017/11/21.
//  Copyright © 2017年 Artup. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        LQAliPay.pay(order: "order info string", appScheme: "app scheme", success: { (result) in
            // 支付成功
        }) {
            // 支付失败
        }
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

